src/Application/Asset/SearchAssetsService.php

```
@@ -0,0 +1,71 @@
+                files: [],
+                totalCount: 0,
+                pageInfo: $pageInfo,
+                userErrors: [new UserError(self::EMPTY_QUERY_CODE, self::EMPTY_QUERY_MESSAGE, 'query')],
+            );
+        }
+
+        $accountId = new AccountId($query->accountId);
+
+        try {
+            $totalCount = $this->assets->countByFileName($accountId, $trimmedQuery, AssetStatus::UPLOADED);
+            $pageInfo = SearchAssetsPageInfo::fromTotalCount($query->page, $query->pageSize, $totalCount);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Application\Asset\Result\SearchAssetsPageInfo' in method 'searchAssets'. (undefined)

(StaticAccess)